alloc.c

```diff
@@ -571,6 +571,58 @@ static void log_reuse(struct liftoff_output *output)
 	output->alloc_reuse++;
 }
 
+static void log_plane_type_change(uint32_t base, uint32_t cmp)
```

Comment:
Not a fan of this commit. When a lot of planes are present most will have the "overlay" type. IMHO plane IDs should be enough (and drm_info can help if more info is needed).

Reply:
Maybe I should just in general remove logging the planes being reset. It does not really give useful information besides a single output line with the ids of all planes on the system.